x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/schema.ts

```diff
@@ -129,5 +135,235 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
         expect(response.body.documentsWithRuntimeFieldsApplied).to.be(null);
       });
     });
+
+    describe('Schema updates via PUT /api/streams/{name}/_ingest', () => {
+      const CLASSIC_STREAM_NAME = 'logs-classic-schema-test';
+      const WIRED_STREAM_NAME = 'logs.wired-schema-test';
+
+      before(async () => {
+        // Create a classic stream by indexing to a hyphen-separated data stream name
+        await indexDocument(esClient, CLASSIC_STREAM_NAME, {
+          '@timestamp': '2024-01-01T00:00:00.000Z',
+          message: 'test document for classic stream',
+        });
+
+        // Create a wired stream by forking from logs
+        await forkStream(apiClient, 'logs', {
+          stream: { name: WIRED_STREAM_NAME },
+          where: { always: {} },
+          status: 'enabled' as RoutingStatus,
+        });
+      });
+
+      after(async () => {
+        await deleteStream(apiClient, WIRED_STREAM_NAME);
+        await deleteStream(apiClient, CLASSIC_STREAM_NAME);
+      });
+
+      it('Updates field mappings for classic streams', async () => {
```

Comment:
There are some tests for this already here: https://github.com/elastic/kibana/blob/eeac35b9ec96af027b5b2edfc104f72c8ef3c368/x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/classic.ts#L318

Reply:
Thanks for noticing that. But don't we wanna test the `/_ingest` endpoint separately? I've only could find the following instance where `/_ingest` is tested but for permissions.
https://github.com/elastic/kibana/blob/cfbde3ad86fadc5378bb36a68fefc852c8087cb1/x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/permissions.ts#L82